libs/core/src/lib/tabs/tab-list.component.html

```diff
@@ -13,7 +13,7 @@
                  [attr.aria-selected]="tab.active">
                 <a
                     fd-tab-link
-                    [attr.tabindex]="0"
+                    [attr.tabindex]= "tab.active ? '0':'-1'"
```

Comment:
Tabbing not possible anymore.

Reply:
Yes that's the requirement. here we have to tab using left and right arrow key. Please refer below link for more details.
https://www.w3.org/TR/wai-aria-practices-1.1/examples/tabs/tabs-2/tabs.html